plugins/outputs/cloudrun/README.md

```diff
@@ -0,0 +1,35 @@
+### Cloud Run Output Plugin
+
+ This plugin is used to send time series metrics into a proxy, located behind the Google Cloud Run service, with OAuth2 enabled.         
```

Comment:
Looking at the bug this fixes, it seems the objective is to send data from a source to a Google CloudRun hosted Wavefront proxy? Does this work with any proxy in cloud run or only wavefront?

Reply:
This should work for any other proxy running in Cloud Run. Our particular use case is Wavefront. The main objective of the Cloud Run output plugin is to handling OAuth 2.0 requirements.